part-4/modularization-concept.md

```diff
@@ -0,0 +1,241 @@
+
+## Commit/update changes
+
+The workflow of commiting changes from a simulation to a building block and updating a simulation with changes from a building block has been changed.
+
+### Updating a simulation with changes from a building block
+
+If a building block used in a simulation is changed after the creation of the simulation, the simulation will automatically be marked as "outdated". To update the simulation with the changes from the building block, simply right-click on the simulation and select "Update from Building Blocks". This will invoke re-creating the simulation from the building blocks, applying any changes made to **all** building blocks since the simulation was created. Any changes to parameter or start values made in the simulation will be lost!
+
+It is no longer possible to update only selected building blocks.
+
+### Committing changes from a simulation to a building block
+
+Changes to parameter or start values made in a simulation can be committed to the building block (BB) the parameter/start value belongs to. To do so, right-click on the simulation and select "Commit to building blocks".
+
+Changes to parameters values are committed to the selected Parameter Values (PV) building block of the last module in the simulation configuration. If the last module does not contain a PV BB, or no PV BB has been selected for this module, new PV BB will be created.
+
+{% hint style="warning" %}
+When committing changes made to a simulation created from PK-Sim module(s) only, the last PK-Sim module will be converted to an extension module. To avoid this, create a new extension module that should contain the changes. Then create the simulation with the extension module.
+{% endhint %}
+
+Changes to initial conditions (IC) of molecules are handled similarly. They are committed to the selected IC BB of the last module in the simulation configuration. If the last module does not contain an IC BB, or no IC BB has been selected for this module, a new IC BB will be created.
+
+Structural changes made to building blocks (e.g., adding/removing reactions, molecules, compartments, etc.) cannot be reverted by committing an "outdated" version of the building blocks from a simulation. Only changes to parameter values and initial conditions can be committed.
+
+## Best practices
+
+The OSP software is best suited for the development of complex quantitative systems pharmacology/toxicology (QSP/T) models based on the physiologically-based kinetics (PBK) modeling framework. With the introduction of the modularization concept, development of such models is even more efficient, transparent, and sustainable. To get the most out of the new concept, the following best practices should be considered.
+
+- If possible, each compound should be represented as a separate PK-Sim module.
```

Comment:
Not sure about this. If we e.g. setup a DDI-Simulation, it will contain several compounds and the user should not try to split them into different modules.

Reply:
Rephrased, please check